src/otx/algo/classification/dino_v2.py

```diff
@@ -48,11 +50,26 @@ def __init__(
     ):
         super().__init__()
         self._init_args = get_class_initial_arguments()
+
+        ci_data_root = os.environ.get("CI_DATA_ROOT")
+        pretrained: bool = True
+        if ci_data_root is not None and Path(ci_data_root).exists():
+            pretrained = False
+
         self.backbone = torch.hub.load(
             repo_or_dir="facebookresearch/dinov2",
             model=backbone,
+            pretrained=pretrained,
         )
 
+        if ci_data_root is not None and Path(ci_data_root).exists():
+            ckpt_filename = f"{backbone}4_pretrain.pth"
+            ckpt_path = Path(Path(ci_data_root) / "torch" / "hub" / "checkpoints" / ckpt_filename)
```

Comment:
```suggestion
            ckpt_path = Path(ci_data_root) / "torch" / "hub" / "checkpoints" / ckpt_filename
```
`Path() / "string"` is also `Path`

Reply:
fixed at https://github.com/openvinotoolkit/training_extensions/pull/3541/commits/b719ffe8faca2a87fa20be15ff3ea26bdcf3648b